multi-node/aws/pkg/config/templates/stack-template.json

```diff
@@ -615,7 +615,7 @@
     ,
     "{{$subnetLogicalName}}RouteTableAssociation": {
       "Properties": {
-        "RouteTableId": "{{.RouteTableID}}",
+        "RouteTableId": "{{$.RouteTableID}}",
```

Comment:
This is inside a `with` block, and consequently needs to use `$.` to access the global scope. Only arises when `routeTableId` cluster.yaml option is set, which we do not test.